releasenotes/notes/min-k8-ver-for-1.9.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+issue:
+- 30176
+releaseNotes:
+- |
+  **Added** Istio 1.9 supports kubernetes versions 1.17 to 1.20.
```

Comment:
Capitalize "Kubernetes", as it is a trademark.

Reply:
Done. Thanks.